.github/actions/gh-pages-storybook/action.yml

```diff
@@ -0,0 +1,6 @@
+name: 'gh-pages-storybook'
```

Comment:
Do we want this action to be in the Canopy repo? I would have probably created a separate repo for it within `Legal-and-General`? you never know that someone else might want to use it?

Reply:
I wanted to trial it here first for speed, it's relatively bespoke to our needs. While it's a local action we don't have to worry about versioning, package creation, linters, project setup etc. There are already some open source actions out there that do this, but I had some hesitancy while there isn't much security tooling around those actions.

I have modified the name though `name: '@legal-and-general/gh-pages-storybook'`